torchbenchmark/models/opacus_cifar10/__init__.py

```diff
@@ -29,7 +31,13 @@ def __init__(self, test, device, batch_size=None, extra_args=[]):
         )
 
         self.model = models.resnet18(num_classes=10)
+        prev_wo_envvar = os.environ.get("TORCH_FORCE_NO_WEIGHTS_ONLY_LOAD", None)
```

Comment:
We don't have access to the `torch.load` call within opacus here, updating opacus also won't help because torchbench doesn't depend on a nightly of opacus but rather stable release